eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,6 +1,4 @@
 variables:
```

Comment:
we need an OS image with Java1.8, the android tools requires java 8. I think below changes needs to be revered.